lib/git-plus.coffee

```diff
@@ -77,11 +74,9 @@ setDiffGrammar = ->
     baseGrammar = baseLineGrammar
 
   if enableSyntaxHighlighting
+    while atom.grammars.grammarForScopeName 'source.diff'
```

Comment:
[maybe not](https://github.com/atom/first-mate/blob/v5.0.2/src/grammar-registry.coffee#L64), but this code was already there.